src/core/bitop.d

```diff
@@ -276,15 +291,21 @@ version (AnyX86)
  * They may be used to guarantee a write or read cycle occurs at a specified address.
  */
 
-ubyte  volatileLoad(ubyte * ptr);
-ushort volatileLoad(ushort* ptr);  /// ditto
-uint   volatileLoad(uint  * ptr);  /// ditto
-ulong  volatileLoad(ulong * ptr);  /// ditto
+pragma(DigitalMars_intrinsic, "ind")
```

Comment:
Having a namespace for them could be a possibility.  But I would consider ``volatileLoad`` and ``volatileStore`` common between all compilers - unlike ``y2lx`` or ``cmpxchg`` - and so will need a meaningful name.